auth-legacy/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+ext {
+    package_id = 'BazaarAuth'
+    artifact_id = 'auth-legacy'
+}
+
+android {
+    compileSdkVersion 29
```

Comment:
Move this variable in another file to reuse it in another Gradle file

Reply:
Done